packages/vite/src/node/plugins/asset.ts

```diff
@@ -287,11 +288,15 @@ async function fileToBuiltUrl(
 
   const file = cleanUrl(id)
   const content = await fsp.readFile(file)
+  const assetsFilter = config.build.assetsInlineExclude
+    ? createFilter(config.build.assetsInlineExclude)
+    : () => false
```

Comment:
I think `createFilter` mean is `true` pass, `false` no pass.

so it feels better

```ts
 const assetsFilter = config.build.assetsInlineExclude
    ? createFilter([], config.build.assetsInlineExclude)
    : () => true
```